backend/controller/pubsub/integration_test.go

```diff
@@ -198,7 +198,7 @@ func TestLeaseFailure(t *testing.T) {
 		in.Sleep(time.Second*7),
 
 		// confirm that the first event failed and the second event succeeded,
-		in.QueryRow("ftl", `SELECT state, error FROM async_calls WHERE verb = 'slow.consume' ORDER BY created_at`, "error", "async call lease expired"),
```

Comment:
I feel like this change would make it harder to figure out why the lease failed... "context canceled" is very generic, vs. "async call lease expired" being very specific